docs/operation/cluster_monitoring.rst

```diff
@@ -0,0 +1,240 @@
+    +-------------------------------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
+    | PrometheusDuplicateTimestamps                   | warning      | Prometheus is dropping samples with duplicate timestamps.                                                           |
+    +-------------------------------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
+    | PrometheusOutOfOrderTimestamps                  | warning      | Prometheus drops samples with out-of-order timestamps.                                                              |
+    +-------------------------------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
+    | PrometheusRemoteStorageFailures                 | warning      | Prometheus fails to send samples to remote storage.                                                                 |
+    +-------------------------------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
+    | PrometheusRemoteWriteBehind                     | warning      | Prometheus remote write is behind.                                                                                  |
+    +-------------------------------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
+    | PrometheusRemoteWriteDesiredShards              | warning      | Prometheus remote write desired shards calculation wants to run more than configured max shards.                    |
+    +-------------------------------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
+    | PrometheusRuleFailures                          | critical     | Prometheus is failing rule evaluations.                                                                             |
+    +-------------------------------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
+    | PrometheusMissingRuleEvaluations                | warning      | Prometheus is missing rule evaluations due to slow rule group evaluation.                                           |
+    +-------------------------------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
```

Comment:
I think it should be automatically generated from the json.

Reply:
Done